docs/root/modules/semgrep/schema.md

```diff
@@ -0,0 +1,91 @@
+
+| Field | Description |
+|-------|--------------|
+| firstseen | Timestamp of when a sync job first discovered this node  |
+| lastupdated | Timestamp of the last time the node was updated |
+| **id** | A composed id based using the repository path and the rule that triggered the finding |
+| **rule_id** | The rule that triggered the finding |
+| **repository** | The repository path where the finding was discovered |
+| summary | A short title summarizing of the finding |
+| description | Description of the vulnerability. |
+| package_manager | The ecosystem of the dependency where the finding was discovered (e.g. pypi, npm, maven) |
+| severity | Severity of the finding (e.g. CRITICAL, HIGH, MEDIUM, LOW) |
+| cve_id | CVE id of the vulnerability from NVD. Check [cve_schema](../cve/schema.md) |
+| reachability_check | Whether the vulnerability reachability is confirmed, not confirmed or needs to be manually confirmed |
+| reachability_condition | Description of the reachability condition (e.g. reachable if code is used in X way) |
+| reachability | Whether the vulnerability is reachable or not |
+| transitivity | Whether the vulnerability is transitive or not (e.g. dependency, transitive) |
+| dependency | Dependency where the finding was discovered. Includes dependency name and version |
+| dependency_fix | Dependency version that fixes the vulnerability |
+| ref_urls | List of reference urls for the finding |
+| dependency_file | Path of the file where the finding was discovered (e.g. lock.json, requirements.txt) |
+| dependency_file_url | URL of the file where the finding was discovered |
+| scan_time | Date and time when the finding was discovered in UTC |
+
+
+#### Relationships
+
+- An SemgrepSCAFinding connected to a GithubRepository (optional)
+
+    ```
+    (SemgrepSCAFinding)-[FOUND_IN]->(GithubRepository)
+    ```
+
+- A SemgrepSCAFinding vulnerable dependency usage at SemgrepSCALocation (optional)
+
+    ```
+    (SemgrepSCAFinding)-[USAGE_AT]->(SemgrepSCALocation)
+    ```
+
+
+### SemgrepSCALocation
+
+Represents the location in a repository where a vulnerable dependency is used in a way that can trigger the vulnerability.
+
+| Field | Description |
+|-------|--------------|
+| firstseen | Timestamp of when a sync job first discovered this node  |
+| lastupdated | Timestamp of the last time the node was updated |
+| **id** | Unique id identifying the location of the finding |
+| path | Path of the file where the usage was discovered |
+| start_line | Line where the usage starts |
+| start_column | Column where the usage starts |
+| end_line | Line where the usage ends |
+| end_column | Column where the usage ends |
```

Comment:
In the NodeSchema, you do the abbreviations end_col and start_col.

Reply:
Autocomplete typo, fixed it.